README.md

```diff
@@ -1,40 +1,229 @@
-# database_based_on_diagram
+<a name="readme-top"></a>
 
-This a Blog app in development with rails and PostgreSQL.
 
-## Setup
+<div align="center">
+  <img src="https://github.com/microverseinc/readme-template/blob/master/murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+</div>
```

Comment:
- [ ] It is not allowed to have **MICROVERSE LOGO** in your README file. Please remove it from your README. However, if you have your own logo, you can use it instead otherwise don't put any logo no problem 😃 👍